setup.py

```diff
@@ -57,12 +57,13 @@
     'mypy == 0.910',
     'pycodestyle >= 2.8.0, < 2.9.0',
     'pylint == 2.10.2',
+    'pytest-parallel < 1.0.0',
```

Comment:
The current version is `0.1.1`. According to SemVer, this means [no warranties are made](https://semver.org/#spec-item-4) on the API. I am thus in favour of locking the minor version.
```suggestion
    'pytest-parallel < 0.2.0',
```